tests/foreman/cli/test_capsule.py

```diff
@@ -17,376 +17,30 @@
-        Docker] etc.
-
-    :Steps: attempt to subscribe a system to a content type variation, via
-        a proxy-enabled capsule
-
-    :expectedresults: system is successfully subscribed to each content
-        type
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-def test_positive_consume_content():
-    """User can consume content on system, from a content source,
-    through proxy-enabled capsule
-
-    :id: a3fb9879-7799-4743-99a8-963701e687c1
-
-    :Setup: Content source types configured/synced for [RH, Custom, Puppet,
-        Docker] etc.
-
-    :Steps:
-
-        1. attempt to subscribe a system to a content type variation, via a
-           proxy-enabled capsule
-        2. Attempt to install content RPMs via
-           proxy-enabled capsule
-
-    :expectedresults: system successfully consume content
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-def test_positive_unsubscribe():
-    """User can unsubscribe system from content through
-    proxy-enabled capsule
-
-    :id: 0d34713d-3d60-4e5a-ada6-9a24aa865cb4
-
-    :Setup: Content source types configured/synced for [RH, Custom, Puppet]
-        etc.
-
-    :Steps:
-
-        1. attempt to subscribe a system to a content type variation, via a
-           proxy-enabled capsule
-        2. attempt to unsubscribe a system from said content type(s) via a
-           proxy-enabled capsule
-
-    :expectedresults: system is successfully unsubscribed from each content
-        type
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-def test_positive_reregister_with_capsule_cert():
-    """system can register via capsule using cert provided by
-    the capsule itself.
-
-    :id: 785b94ea-ffbf-4c18-8160-f705e3d7cbe6
-
-    :Setup: functional capsule and certs rpm installed on target client.
-
-    :Steps:
-
-        1. Attempt to register from parent satellite; unregister and remove
-           cert rpm
-        2. Attempt to reregister using same credentials and certs from a
-           functional capsule.
-
-    :expectedresults: Registration works , and certs RPM installed from
-        capsule.
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-def test_positive_ssl_capsule():
-    """Assure SSL functionality for capsules
-
-    :id: 4d19bee6-15d4-4fd5-b3de-9144608cdba7
-
-    :Setup: A capsule installed with SSL enabled.
-
-    :Steps: Execute basic steps from above (register, subscribe, consume,
-        unsubscribe, unregister) while connected to a capsule that is
-        SSL-enabled
-
-    :expectedresults: No failures executing said test scenarios against
-        SSL, baseline functionality identical to non-SSL
-
-    :CaseAutomation: NotAutomated
-    """
-
-
 @pytest.mark.stubbed
-def test_positive_enable_bmc():
-    """Enable BMC feature on smart-proxy
+@pytest.mark.e2e
+def test_positive_capsule_content():
```

Comment:
Add upgrade mark please.

Reply:
wdyt ?

Reply:
hello, yes I added it, somehow github doesn't mark the the conversation as outdated